apps/web/public/static/locales/en/common.json

```
@@ -1599,5 +1599,23 @@
   "booking_with_payment_cancelled_already_paid": "A refund for this booking payment it's on the way.",
   "booking_with_payment_cancelled_refunded": "This booking payment has been refunded.",
   "booking_confirmation_failed": "Booking confirmation failed",
-  "get_started_zapier_templates": "Get started with Zapier templates"
+  "get_started_zapier_templates": "Get started with Zapier templates",
+  "a_routing_form": "A Routing Form",
+  "form_description_placeholder": "Form Description",
+  "keep_me_connected": "keep me connected with the form",
```

`keep_me_connected` is more generic but the text is more specific.
```suggestion
  "keep_me_connected_with_form": "keep me connected with the form",
```

---

Also, the text was starting with K capital